src/crawler/crawlers/arso_water_slovenija/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "arso_data",
```

Comment:
This is not a standalone crawler, but part of crawlers. Implement it like this. Remove package.json from here.